app/workers/expired_rent_scheduler.rb

```diff
@@ -0,0 +1,13 @@
+require 'sidekiq-scheduler'
+require 'date'
+
+class ExpiredRentScheduler
+  include Sidekiq::Worker
+
+  def perform
+    rents = Rent.where('final_date = ?', Time.zone.today)
```

Comment:
You can write it like that
```suggestion
    rents = Rent.where(final_date: Time.zone.today)
```